### ISSUE TITLE
NTR: PR3 (proteinase 3) antigen positive (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8788]
New term request PR3 (proteinase 3) antigen positive (segal_210427183623): Corresponds to C-ANCA; antineutrophil cytoplasmic antigen

**Comment by pnrobinson** [comment]
@MickeySegal
this is 
[Anti-proteinase 3 antibody positivity HP:0033557](https://hpo.jax.org/app/browse/term/HP:0033557)

I think the label should be anti-PR3 antibody positive (not antigen as stated above)

**Comment by MickeySegal** [comment]
Anti-PR3 antibodies and PR3 antigen positive are equivalent statements from different perspectives; I'm glad to use either.
